### PR TITLE
chore: bump to node 20

### PR DESCRIPTION
**Pull request by rsre**
Node 16 has reached its end of life, prompting [GitHub to initiate its deprecation process for GitHub Actions](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/).

This PR bumps the node version to 20 to keep up with GitHub runner's.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16.19.0
+20.11.0
```
Comment (rsre) [inline .nvmrc:1]: I chose 20.11.0 as that is the latest LTS by Node.js https://nodejs.org/en/blog/release/

**Review by shivamkumar94 — APPROVED** [review]

**Comment by mblumenstingl-n11** [comment]
@shioyang any plans to merge this and create a new release?

**Comment by shioyang** [comment]
Thank you for the PR.
I just released the latest version with the latest node version. Would you try it?

**Comment by mblumenstingl-n11** [comment]
@shioyang unfortunately it does not work for me yet.
43643554762ed3e9cb4991ad368891276d080b4c only has half of the changes suggested by @rsre: it's missing the update to `action.yml` (which in the master branch still references `node16`)

**Comment by mblumenstingl-n11** [comment]
you can also see it for your own repo: https://github.com/shioyang/check-pr-labels-on-push-action/actions/runs/8358038388
the `Annotations` section shows:
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: shioyang/check-pr-labels-on-push-action@v1.0.10. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

**Comment by shioyang** [comment]
@mblumenstingl-n11 Thank you for your comments. fixed!

**Pull request closed without merging**